README.md

```diff
@@ -404,4 +405,4 @@ django-todo no longer references a jQuery datepicker, but defaults to native htm
 
 ### URLs
 
-Some views and URLs were renamed for logical consistency. If this affects you, see source code and the demo GTD site for reference to the new URL names.
\ No newline at end of file
+Some views and URLs were renamed for logical consistency. If this affects you, see source code and the demo GTD site for reference to the new URL names.
```

Comment:
Weird, I can't for the life of me spot the difference in these two lines! Ah well.